src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.gpiod_line_request_both_edges_events.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+using System;
+using System.Device.Gpio;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    [DllImport(LibgpiodLibrary, SetLastError = true)]
+    internal static extern int gpiod_line_request_both_edges_events(SafeLineHandle line, string consumer);
```

Comment:
Does it make sense to put each of these in their own file? I get it for completely different API, but these all seem similar, in the same library, etc.

Reply:
I don't like it too, but libc library is written this way, i thought it is the convention